AdminWebsite/AdminWebsite/ClientApp/src/app/testing/data/work-allocation-test-data.ts

```diff
@@ -6,108 +6,108 @@ export class MockWorkAllocationValues {
          [
             {
                "id":1,
-               "end_time":"2023-01-01T15:00:00Z",
```

Comment:
Please revert this change. After you implement @will-craig 's suggestion to use a filter variable you can adjust the test setup to minimise changes

Reply:
done